cmd/gossamer/configcmd.go

```diff
@@ -267,20 +244,3 @@ func dumpConfig(ctx *cli.Context) error {
 	}
 	return nil
 }
-
-// These settings ensure that TOML keys use the same names as Go struct fields.
-var tomlSettings = toml.Config{
-	NormFieldName: func(rt reflect.Type, key string) string {
-		return key
-	},
-	FieldToKey: func(rt reflect.Type, field string) string {
-		return field
-	},
-	MissingField: func(rt reflect.Type, field string) error {
-		link := ""
-		if unicode.IsUpper(rune(rt.Name()[0])) && rt.PkgPath() != "main" {
-			link = fmt.Sprintf(", see https://godoc.org/%s#%s for available fields", rt.PkgPath(), rt.Name())
-		}
-		return fmt.Errorf("field '%s' is not defined in %s%s", field, rt.String(), link)
-	},
-}
```

Comment:
I am fine removing this but we still require some validation on the passed in toml file. Misspellings such as `Por` results in using the default settings without any indication to the user.

Reply:
The new library has no explicit way to detect unprocessed keys unfortunately, although it was discussed here: https://github.com/pelletier/go-toml/issues/277

Reply:
I am a bit torn on this and I feel we need some way of indicating to users when their configuration within the `toml` file is not being used due to an error. The toml parsing into config is case sensitive, so when a user indicates the RPC modules like so:

```
[rpc]
Modules=["author] // instead of modules=["author"]
``` 
This is not captured as an error, the default RPC modules are used instead but the user has no indication of this. The same applies to all fields.

Is the interface of this new toml library that much better that we want to forgo notifying user-specified configuration errors?

Reply:
Reverted to the old parser